components/FlexibleHeader/src/MDCFlexibleHeaderView.m

```diff
@@ -1140,7 +1140,7 @@ - (void)fhv_contentOffsetDidChange {
       // focusing.
       CGPoint offset = self.trackingScrollView.contentOffset;
       offset.y = MAX(offset.y, -self.maximumHeight);
-      self.trackingScrollView.contentOffset = offset;
```

Comment:
So will the scrollview's contentOffset be mismatched with what the FHV thinks it is after this? If the user turns off VoiceOver after scrolling, is the FHV going to be offset incorrectly or anything like that?

Reply:
The FHV always reads the scroll view’s content offset directly, so changing it like so here won’t affect the FHV’s future readings.

Reply:
Thanks! I guess I was confused because it looked like there were 2 different mechanisms for updating the FHV's contentOffset.